_inc/client/searchable-modules/index.jsx

```diff
@@ -38,12 +39,16 @@ export const SearchableModules = withModuleSettingsFormHelpers(
 			}
 
 			// Only should be features that don't already have a UI, and we want to reveal in search.
-			const whitelist = [ 'contact-form', 'enhanced-distribution', 'json-api', 'notes' ];
+			const whitelist = [ 'contact-form', 'enhanced-distribution', 'json-api', 'notes', 'private' ];
 
 			const allModules = this.props.modules,
 				results = [];
 			forEach( allModules, ( moduleData, slug ) => {
 				if ( this.props.isModuleFound( slug ) && includes( whitelist, slug ) ) {
+					// The Private Sites gets its own card.
```

Comment:
I chose to go that route since the other searchable modules in the whitelist use a basic card without a toggle to turn the feature off. We need such a toggle for our module.